server/routerlicious/packages/lambdas/src/test/scribe/lambda.spec.ts

```diff
@@ -0,0 +1,137 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// import { IGitManager } from "@fluidframework/server-services-client";
+import { ICreateTreeEntry, ICreateTreeParams, ITree } from "@fluidframework/gitresources";
+import { GitManager } from "@fluidframework/server-services-client";
+import { DefaultServiceConfiguration, ICollection, IDocument, IProducer, ITenantManager, MongoManager } from "@fluidframework/server-services-core";
+import { KafkaMessageFactory, MessageFactory, TestCollection, TestContext, TestDbFactory, TestKafka, TestTenantManager } from "@fluidframework/server-test-utils";
+import { strict as assert } from "assert";
+import _ from "lodash";
+import nconf from "nconf";
+import { ScribeLambda } from "../../scribe/lambda";
+import { ScribeLambdaFactory } from "../../scribe/lambdaFactory";
+
+describe("Routerlicious", () => {
+    describe("Scribe", () => {
+        describe("Lambda", () => {
+            const testClientId = "test";
+            const testTenantId = "test";
+            const testDocumentId = "test";
+
+            let testMongoManager: MongoManager;
+            let testDocumentCollection: ICollection<IDocument>;
+            let testMessageCollection: TestCollection;
+            let testProducer: IProducer;
+            let testContext: TestContext;
+            let testTenantManager: ITenantManager;
+            let testKafka: TestKafka;
+            let messageFactory: MessageFactory;
+            let kafkaMessageFactory: KafkaMessageFactory;
+            let lambda: ScribeLambda;
+            let testGitManager: GitManager;
+            let tree : ITree; 
+
+            function sendOps(num: number): void {
```

Comment:
Would it be useful to move this to an utils folder? I see that it is being used in scriptorium and broadcaster too.